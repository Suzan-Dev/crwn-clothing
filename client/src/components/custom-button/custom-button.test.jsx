import React from 'react';
import { shallow } from '../../setupTests';
import CustomButton from './custom-button.component';

describe('Testing: CustomButton component!', () => {
  const wrapper = shallow(<CustomButton />);

  test('should render CustomButton component.', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
