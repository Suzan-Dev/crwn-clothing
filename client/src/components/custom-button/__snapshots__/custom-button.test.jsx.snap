// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing: CustomButton component! should render CustomButton component. 1`] = `"<styled.button />"`;
